patches/server/0863-Fix-OfflinePlayer-getBedSpawnLocation.patch

```diff
@@ -39,8 +49,38 @@ index d7823d7dc88cfba6f6ac9dae220e03dea4a0bcdd..6d2ba650f53de8a460857f1846401a20
 +            if (spawnWorld == null) {
 +                return null;
 +            }
-+            return new Location(spawnWorld, data.getInt("SpawnX"), data.getInt("SpawnY"), data.getInt("SpawnZ"), respawnAngle, 0);
++            if (!load) {
```

Comment:
instead of exiting early, it might be possible to adapt the findRespawnPositionAndUseSpawnBlock method to not load anything not already loaded. Then if the area being searched is already loaded, the full logic can run even if the load param is false

Reply:
That's so true, thanks